packages/x-components/src/components/animations/animate-width.vue

```diff
@@ -34,7 +34,7 @@
 </style>
```

Comment:
Leaving it here because I don't know where to put it. `redirection.vue` has a header `### Play with the component` that should be `## Play with the component`.